src/phase0/state_transition/block_processing.rs

```diff
@@ -236,16 +236,28 @@ fn process_eth1_data<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _body: &BeaconBlockBody<
+    body: &BeaconBlockBody<
         MAX_PROPOSER_SLASHINGS,
         MAX_VALIDATORS_PER_COMMITTEE,
         MAX_ATTESTER_SLASHINGS,
         MAX_ATTESTATIONS,
         MAX_DEPOSITS,
         MAX_VOLUNTARY_EXITS,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
```

Comment:
Should I remove `Result<(), Error>` from the function return type if no error can be returned?

Reply:
yes please!

Reply:
Done: da0a232